src/Symfony/Component/Security/Core/SecurityContext.php

```diff
@@ -60,11 +60,7 @@ public function __construct(AuthenticationManagerInterface $authenticationManage
             $this->token = $this->authenticationManager->authenticate($this->token);
         }
 
-        if (!is_array($attributes)) {
-            $attributes = array($attributes);
-        }
-
-        return $this->accessDecisionManager->decide($this->token, $attributes, $object);
+        return $this->accessDecisionManager->decide($this->token, (array) $attributes, $object);
```

Comment:
Direct parameter casting to array in Symfony/Component/Security/Core/SecurityContext.php:63